pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -42,74 +46,96 @@ import (
 	samplescheme "github.com/xmudrii/etcdproxy-controller/pkg/client/clientset/versioned/scheme"
 	informers "github.com/xmudrii/etcdproxy-controller/pkg/client/informers/externalversions/etcd/v1alpha1"
 	listers "github.com/xmudrii/etcdproxy-controller/pkg/client/listers/etcd/v1alpha1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-const controllerAgentName = "etcdstorage-controller"
+const controllerAgentName = "etcdproxy-controller"
```

Comment:
user agent for http? better call it like that

Reply:
Is `httpUserAgentName` okay?

Reply:
👍